database/buildSchema/30_review_status_history.sql

```diff
@@ -0,0 +1,37 @@
+-- review section join
+
+
+CREATE TYPE public.review_status as ENUM ('Draft', 'Submitted', 'Changes Required', 'Review Pending');
+
+CREATE TABLE public.review_status_history (
+	id serial primary key,
+	review_id integer references public.review(id),
+	"status" public.review_status,
+	"timestamp" timestamp with time zone,
+	is_current boolean DEFAULT true
+);
+
+-- FUNCTION to set `is_current` to false on all other review_status_histories of current review
+CREATE OR REPLACE FUNCTION public.review_status_history_is_current_update()
+RETURNS trigger as $review_status_history_event$
+BEGIN
+	UPDATE public.review_status_history SET is_current = false
+	WHERE review_id = NEW.review_status_history AND id<>NEW.id;
```

Comment:
`NEW.review_status_history` refers to a field, not a table, and there's no `review_status_history` field. (It's throwing an error when I try to INSERT a row into this table). Isn't it just:
`WHERE review_id = NEW.review_id AND id<>NEW.id;`

Reply:
thanks for catching that, it's defo a typo:
b4cbe4e1c413055396043afb9e5080e19a46e022